PubnativeLite/PubnativeLite/Viewability/HyBidViewabilityAdSession.m

```diff
@@ -52,10 +52,14 @@ - (void)fireOMIDImpressionOccuredEvent:(OMIDPubnativenetAdSession *)omidAdSessio
         return;
     
     if(omidAdSession != nil){
-        NSError *adEventsError;
-        OMIDPubnativenetAdEvents *adEvents = [[OMIDPubnativenetAdEvents alloc] initWithAdSession:omidAdSession error:&adEventsError];
+        if ([HyBidViewabilityManager sharedInstance].adEvents == nil) {
+            NSError *adEventsError;
+            [HyBidViewabilityManager sharedInstance].adEvents = [[OMIDPubnativenetAdEvents alloc] initWithAdSession:omidAdSession error:&adEventsError];
```

Comment:
Won't this be a problem if we have more than one viewability session open at the same time? Like two placements on the same screen?

Reply:
Check my last commit. Now the adEvents is linked to an adSession.